### ISSUE TITLE
Please clarify and/or document Openstack support

### ISSUE THREAD

**Issue by greenmoss** [infochimps-labs/ironfan#333]
The ironfan Github repository description includes "Provision EC2, OpenStack or Vagrant". However the documentation only mentions EC2:

https://github.com/infochimps-labs/ironfan/wiki/INSTALL

I also see that an openstack pull request opened just today:

https://github.com/infochimps-labs/ironfan/pull/332

From these two clues, I'm guessing OpenStack isn't well supported. Is this accurate?

If so, maybe take "Openstack" out of the Github repository description?

If not, is there Openstack documentation?

**Comment by howech** [comment]
You are correct - the openstack support is not documented - before my pull request, you probably had to fake things up by having ironfan talk to openstack through its ec2-emulation api. Even with my pull request, OpenStack is still not well supported, as there are many things in openstack that you can not get to through ironfan cluster definitions. It is likely that this will change a little over the next couple of months.
